Farmhash.Sharp.Benchmarks/BaseConfig.cs

```diff
@@ -11,29 +11,31 @@ public class BaseConfig : ManualConfig
     {
         public BaseConfig()
         {
-            Add(new CsvExporter(CsvSeparator.CurrentCulture,
-                new BenchmarkDotNet.Reports.SummaryStyle
-                {
-                    PrintUnitsInContent = false,
-                    TimeUnit = BenchmarkDotNet.Horology.TimeUnit.Nanosecond
-                }));
+            Add(new CsvExporter(
+                CsvSeparator.CurrentCulture,
+                new BenchmarkDotNet.Reports.SummaryStyle(
+                    false,
+                    null,
+                    BenchmarkDotNet.Horology.TimeUnit.Nanosecond,
+                    false)
+            ));
 
             Add(StatisticColumn.Mean);
             Add(StatisticColumn.StdErr);
             Add(StatisticColumn.StdDev);
             Add(StatisticColumn.Median);
 
             // dotnet cli toolchain supports only x64 compilation
-            Add(new Job("core-64bit", EnvironmentMode.Core)
+            Add(new Job("core-64bit").With(CoreRuntime.Core21).WithId("Core")
                 .With(CsProjCoreToolchain.NetCoreApp21));
 
-            Add(new Job("net-legacy-32bit", EnvironmentMode.LegacyJitX86));
-            Add(new Job("net-legacy-64bit", EnvironmentMode.LegacyJitX64));
+            Add(new Job("net-legacy-32bit", EnvironmentMode.LegacyJitX86).With(CsProjClassicNetToolchain.Net48));
+            Add(new Job("net-legacy-64bit", EnvironmentMode.LegacyJitX64).With(CsProjClassicNetToolchain.Net48));
```

Comment:
With our discussion in https://github.com/nickbabcock/Farmhash.Sharp/pull/23, I would suspect that this should target an earlier .net classic toolchain. Also is the `.With` necessary (eg: what happens if it is removed, do the benchmarks fail?)

Reply:
I agree for .net 45 target.
The with is necessary because the framework maintainers changed the default toolchain.
If you remove the .With you get errors at beginning of run saying those test won't be executed.

Reply:
Note the there is no Net45 toolchain. The oldest one is CsProjClassicNetToolchain.Net461

Reply:
Without with, you get lots of messages like that :

```
> Currently dotnet cli toolchain supports only RyuJit, benchmark 'HashBenchmark32.FarmHash: net-legacy-32bit(Jit=LegacyJit, Platform=X86) [PayloadLength=4]' will not be executed
> Currently dotnet cli toolchain supports only RyuJit, benchmark 'HashBenchmark32.Md5: net-legacy-32bit(Jit=LegacyJit, Platform=X86) [PayloadLength=4]' will not be executed
> Currently dotnet cli toolchain supports only RyuJit, benchmark 'HashBenchmark32.StringHashCode: net-legacy-32bit(Jit=LegacyJit, Platform=X86) [PayloadLength=4]' will not be executed
> Currently dotnet cli toolchain supports only RyuJit, benchmark 'HashBenchmark32.SparrowXXHash: net-legacy-32bit(Jit=LegacyJit, Platform=X86) [PayloadLength=4]' will not be executed
> Currently dotnet cli toolchain supports only RyuJit, benchmark 'HashBenchmark32.HFCityHash: net-legacy-32bit(Jit=LegacyJit, Platform=X86) [PayloadLength=4]' will not be executed
> Currently dotnet cli toolchain supports only RyuJit, benchmark 'HashBenchmark32.SpookyHash: net-legacy-32bit(Jit=LegacyJit, Platform=X86) [PayloadLength=4]' will not be executed
> Currently dotnet cli toolchain supports only RyuJit, benchmark 'HashBenchmark32.Spookily: net-legacy-32bit(Jit=LegacyJit, Platform=X86) [PayloadLength=4]' will not be executed
> Currently dotnet cli toolchain supports only RyuJit, benchmark 'HashBenchmark32.XXHash: net-legacy-32bit(Jit=LegacyJit, Platform=X86) [PayloadLength=4]' will not be executed
> Currently dotnet cli toolchain supports only RyuJit, benchmark 'HashBenchmark32.CityHashNet: net-legacy-32bit(Jit=LegacyJit, Platform=X86) [PayloadLength=4]' will not be executed
> Currently dotnet cli toolchain supports only RyuJit, benchmark 'HashBenchmark32.FarmHash: net-legacy-64bit(Jit=LegacyJit, Platform=X64) [PayloadLength=4]' will not be executed
> Currently dotnet cli toolchain supports only RyuJit, benchmark 'HashBenchmark32.Md5: net-legacy-64bit(Jit=LegacyJit, Platform=X64) [PayloadLength=4]' will not be executed
> Currently dotnet cli toolchain supports only RyuJit, benchmark 'HashBenchmark32.StringHashCode: net-legacy-64bit(Jit=LegacyJit, Platform=X64) [PayloadLength=4]' will not be executed
> Currently dotnet cli toolchain supports only RyuJit, benchmark 'HashBenchmark32.SparrowXXHash: net-legacy-64bit(Jit=LegacyJit, Platform=X64) [PayloadLength=4]' will not be executed
> Currently dotnet cli toolchain supports only RyuJit, benchmark 'HashBenchmark32.HFCityHash: net-legacy-64bit(Jit=LegacyJit, Platform=X64) [PayloadLength=4]' will not be executed
> Currently dotnet cli toolchain supports only RyuJit, benchmark 'HashBenchmark32.SpookyHash: net-legacy-64bit(Jit=LegacyJit, Platform=X64) [PayloadLength=4]' will not be executed
> Currently dotnet cli toolchain supports only RyuJit, benchmark 'HashBenchmark32.Spookily: net-legacy-64bit(Jit=LegacyJit, Platform=X64) [PayloadLength=4]' will not be executed
> Currently dotnet cli toolchain supports only RyuJit, benchmark 'HashBenchmark32.XXHash: net-legacy-64bit(Jit=LegacyJit, Platform=X64) [PayloadLength=4]' will not be executed
> Currently dotnet cli toolchain supports only RyuJit, benchmark 'HashBenchmark32.CityHashNet: net-legacy-64bit(Jit=LegacyJit, Platform=X64) [PayloadLength=4]' will not be executed
```